runners/flink/runner/src/main/java/org/apache/beam/runners/flink/FlinkRunnerResult.java

```diff
@@ -86,4 +87,9 @@ public State waitUntilFinish() {
   public State waitUntilFinish(Duration duration) {
     throw new UnsupportedOperationException("FlinkRunnerResult does not support waitUntilFinish.");
   }
+
+  @Override
+  public MetricResults metrics() {
+    throw new UnsupportedOperationException("The Flink Runner does not yet support metrics.");
```

Comment:
Make your wording consistent.

I like "The %sRunner does not currently support metrics", but it's up to you.